packages/cli/src/command-build.js

```diff
@@ -322,7 +323,8 @@ async function addLastBuildTimeToPackageInfos(packageInfos, packageChanges, root
             cwd: rootDirectory,
           })
 
-          if (stdout.includes('[bilt-with-bilt]')) {
+          const branchName = await getGitBranchName(rootDirectory)
```

Comment:
This command is inside the "loop" (it's a `map`, so not really a loop). This means that it gets executed for each packageChange, where it could be executed outside of it all, and just once (as could be the string `[bilt-with-bilt-${branchName}]`, which could be also calculated outside the loop.